omniledger/service/service.go

```diff
@@ -173,10 +173,14 @@ func (s *Service) GetProof(req *GetProof) (resp *GetProofResponse, err error) {
 	log.Lvlf2("%s: Getting proof for key %x on sc %x", s.ServerIdentity(), req.Key, req.ID)
 	latest, err := s.db().GetLatest(s.db().GetByID(req.ID))
 	if err != nil {
+		log.Error(
```

Comment:
Don't print the error here. Following yesterday's discussion: this method _does_ return an `error`, so the caller can display it, if it is needed. Same comment for line 182.

Reply:
Fixed it